packages/perseus/src/util/deprecated-widgets/modernize-widgets-utils.ts

```diff
@@ -41,3 +41,20 @@ export const conversionRequired = (json: PerseusRenderer): boolean => {
 
     return false;
 };
+
+// Convert the user input data keys from input-number to numeric-input
+export const convertUserInputData = (
+    userInputMap: UserInputMap,
+): UserInputMap => {
+    const updatedUserInputMap = {...userInputMap};
+
+    for (const key of Object.keys(userInputMap)) {
+        if (key.includes("input-number")) {
+            const updatedKey = key.replace("input-number", "numeric-input");
```

Comment:
Just realized this is input number specific. I'll move this logic to the appropriate file, and keep this function a little more generic for our future selves.